qos-monitor/src/main/resources/application.properties

```diff
@@ -26,17 +26,17 @@ domain.port=8451
 ############################################
 
 # Name of the core system
-core_system_name=QOSMONITOR
+core_system_name=qosmonitor
 
 # Show all request/response in debug log
-log_all_request_and_response=false
+log_all_request_and_response=true
```

Comment:
should be false when we merge into development

Reply:
done